.eslintrc.js

```diff
@@ -11,6 +11,9 @@ module.exports = {
     babelOptions: {
       configFile: path.resolve(__dirname, './babel.config.json'),
     },
+    project: 'tsconfig.json',
+    tsconfigRootDir: __dirname,
+    sourceType: 'module',
```

Comment:
This change fixed the error:
![image](https://user-images.githubusercontent.com/93188219/209518433-1f3e46fb-4e1e-4cc3-aff3-8a39107d7fd3.png)